binrz/rz-test/run.c

```diff
@@ -214,14 +214,14 @@ RZ_API bool rz_test_cmp_cmd_output(const char *output, const char *expect, const
 			if (expect_len != match_str_len) {
 				return false;
 			}
-			equal = (rz_str_cmp(expect, rz_strbuf_get(match_str), expect_len) == 0);
+			equal = rz_str_cmp(expect, rz_strbuf_get(match_str), expect_len) == 0;
 		} else {
 			equal = RZ_STR_EQ(expect, rz_strbuf_get(match_str));
 		}
 		rz_strbuf_free(match_str);
 		return equal;
 	}
-	return (0 == strcmp(expect, output));
+	return 0 == strcmp(expect, output);
```

Comment:
maybe use `!rz_str_cmp` and `!strcmp`

Reply:
ok 1d74226fe7901c8d7ba979f6da621ccb005ec2f2